common/Subtitles/SbvParser.cs

```diff
@@ -0,0 +1,47 @@
+﻿using StorybrewCommon.Util;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace StorybrewCommon.Subtitles
+{
+    public class SbvParser
+    {
+        public SubtitleSet Parse (String path) 
+        {
+            string[] rawLines = System.IO.File.ReadAllLines(path);
+            List<SubtitleLine> lines = new List<SubtitleLine>();
+
+            double sTime = 0;
+            double eTime = 0;
+            String text = "";
+            foreach (String line in rawLines) 
+            {
+                String[] times = line.Split(','); //Get timing Strings.
```

Comment:
No need to comment everything, put important comments before the line, not on the same one.